bob-apps/bin/bobd.rs

```diff
@@ -250,6 +387,33 @@ fn check_folders(node: &NodeConfig, init_flag: bool) {
 
 fn get_matches<'a>() -> ArgMatches<'a> {
     let ver = format!("{}\n{}", crate_version!(), BuildInfo::default());
+    let testmode_sc = SubCommand::with_name("testmode")
+        .about("Bob's test mode")
+        .arg(
+            Arg::with_name("data")
+            .help("path to bob data")
```

Comment:
`path to bob data directory`

Reply:
Fix: [cdc7316](https://github.com/qoollo/bob/pull/588/commits/cdc73164e2a79a83876565868f58bcedc4659ebf)